package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@axelar-network/axelar-cgp-solidity",
-  "version": "5.0.0",
+  "version": "5.0.1",
   "description": "EVM Smart Contracts for Axelar Network",
   "main": "index.js",
   "scripts": {
-    "build": "hardhat clean && hardhat compile",
+    "build": "hardhat clean && hardhat compile && npm run copy:artifacts",
+    "copy:interfaces": "rm -rf interfaces && mkdir interfaces && cp artifacts/contracts/interfaces/*/*.json interfaces/",
```

Comment:
i think we should stick to `artifacts` for everything. it's more clear to apps how the folder structure looks like